api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/EntitlementCalculator.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.dhsc.htbhf.claimant.service;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import uk.gov.dhsc.htbhf.claimant.entity.Claimant;
+import uk.gov.dhsc.htbhf.claimant.model.eligibility.EligibilityResponse;
+
+@Component
+@Slf4j
+public class EntitlementCalculator {
```

Comment:
Can we have some documentation on the business rules being applied please, either here in JavaDoc or in the README.md?